lib/ExternalModule.js

```diff
@@ -137,6 +137,7 @@ class ExternalModule extends Module {
 			case "amd-require":
 			case "umd":
 			case "umd2":
+			case "system":
```

Comment:
System format is so similar to amd/umd that I was able to entirely reuse the AMD/UMD implementation of externals.